hackathon_site/event/migrations/0005_hss_test_users.py

```diff
@@ -0,0 +1,27 @@
+from django.db import migrations
+from django.contrib.auth.management import create_permissions
+
+
+def apply_migration(apps, schema_editor):
+    for app_config in apps.get_app_configs():
+        app_config.models_module = True
+        create_permissions(app_config, apps=apps, verbosity=0)
+        app_config.models_module = None
+
+    Group = apps.get_model("auth", "Group")
+    Permission = apps.get_model("auth", "Permission")
+
+    group, _ = Group.objects.get_or_create(name="HSS Test Users")
```

Comment:
After I applied the migration, there is no change in my database. I just manually created the group